google/protobuf/duration.proto

```diff
@@ -0,0 +1,104 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Adding protobuf protos into googleapis will cause confusion, as the same protos are already in https://github.com/google/protobuf/tree/master/src/google/protobuf